standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/tools/schematool/TestSchemaToolForMetastore.java

```diff
@@ -370,6 +370,44 @@ public void testSchemaUpgrade() throws Exception {
     schemaTool.verifySchemaVersion();
   }
 
+  /**
+   * Test schema upgrade checking that the alpha version is called
+   */
+  @Test
+  public void testSchemaUpgradeFromAlphaVersion() throws Exception {
+    boolean foundException = false;
+    // Initialize 1.2.0 schema
+    execute(new SchemaToolTaskInit(), "-initSchemaTo 3.1.0");
```

Comment:
nit: comment does not add much to the code thus could be removed. Also the version in the comment is different than the one actually used.

Reply:
removed the comment